standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/IMetaStoreClient.java

```diff
@@ -2716,6 +2719,33 @@ boolean deletePartitionColumnStatistics(String catName, String dbName, String ta
       String partName, String colName, String engine)
       throws NoSuchObjectException, MetaException, InvalidObjectException, TException, InvalidInputException;
 
+  /**
+   * Delete partition level column statistics given dbName, tableName, partName and colName, or
+   * all columns in a partition.
+   * @param dbName database name.
+   * @param tableName table name.
+   * @param partName partition name.
+   * @param colNames a list of column name, or null for all columns
+   * @param engine engine, or null for all engines
+   * @return boolean indicating outcome of the operation
+   * @throws NoSuchObjectException no such partition exists
+   * @throws InvalidObjectException error dropping the stats data
+   * @throws MetaException error accessing the RDBMS
+   * @throws TException thrift transport error
+   * @throws InvalidInputException input is invalid or null.
+   */
+  public boolean deletePartitionMultiColumnStatistics(String dbName, String tableName, String partName,
+                                                      List<String> colNames, String engine) throws TException;
+
+  /**
+   * Delete partition level column statistics given dbName, tableName, partName and colName, or
+   * all columns in a partition.
+   * @param req the DeletePartitionColumnStatisticsRequest which including
+   *            catalog name, database name, table name, partition name, column names, and engine name
+   * @throws TException thrift transport error
+   */
+  public boolean deletePartitionMultiColumnStatistics(DeletePartitionColumnStatisticsRequest req) throws TException;
```

Comment:
I would like to merge the `deletePartitionMultiColumnStatistics` and `deleteTableMultiColumnStatistics`, and keep only one method, i.e, a `deleteMultiColumnStatistics(DeleteColumnStatisticsRequest req)` for both dropping table statistics and partition statistics.